app/src/src/components/containers/SearchResults.jsx

```diff
@@ -0,0 +1,268 @@
+import React, { useState, useEffect, useCallback } from 'react';
```

Comment:
Ive noticed i have quite inconsistent variable names, ill look into this

Reply:
What do you mean? These are built-in functions

Reply:
I just mean in general im all over the place, not specifically here